index.js

```diff
@@ -1 +1 @@
-module.exports = require('./lib/micro-enum.js');
+module.exports=require('./lib/micro-enum.js');
```

Comment:
This is only used on the developer side, right? So the user doesn't download this, if I understand correctly. What do these whitespace removals improve?